### PR TITLE
Make sure that the ogv property exists before using it

### LINKED ISSUE #6524: VideoPress: PHP Notices when fetching information about a video
```
[28-Feb-2017 14:23:27 UTC] PHP Notice:  Undefined property: stdClass::$ogv in /wp-content/plugins/jetpack/modules/videopress/class.videopress-player.php on line 329
[28-Feb-2017 14:23:27 UTC] PHP Notice:  Trying to get property of non-object in /wp-content/plugins/jetpack/modules/videopress/class.videopress-player.php on line 329
```

### PR DESCRIPTION
**Pull request by dbtlr**
Fixes #6524

Note: stdClass sucks.

### CODE REVIEW

File: modules/videopress/class.videopress-player.php [hunk 1]
```diff
@@ -326,10 +326,14 @@ private function html5_static() {
...
 		}
-		$ogg = $this->video->videos->ogv->url;
-		if ( ! empty( $ogg ) )
-			$html .= '<source src="' . esc_url( $ogg ) . '" type="video/ogg; codecs=&quot;' . esc_attr( $this->video->videos->ogv->codecs ) . '&quot;" />';
-		unset( $ogg );
+
+		if ( isset( $this->video->videos->ogv ) ) {
+			$ogg = $this->video->videos->ogv->url;
+			if ( ! empty( $ogg ) ) {
+				$html .= '<source src="' . esc_url($ogg) . '" type="video/ogg; codecs=&quot;' . esc_attr($this->video->videos->ogv->codecs) . '&quot;" />';
```
Comment (zinigor) [inline modules/videopress/class.videopress-player.php]: Sorry for the nitpick, but you have removed spaces in braces here.

Comment (dbtlr) [inline modules/videopress/class.videopress-player.php]: This is all previously written code. If you look at the original version, the whole file is improperly spaced. Are you asking me to respace the whole file?

The only lines I actually added were: `if ( isset( $this->video->videos->ogv ) ) {` and `}`

Comment (dbtlr) [inline modules/videopress/class.videopress-player.php]: PHPStorm removed them when I tabbed the block in, not sure why.

**Comment by dbtlr** [comment]
@briancolinger: I'm not going to debate code style here, but this file was previously written by an old Automattician and the purpose of this pull request is to fix a notice. As of now, beyond this fix, this code is stable. I'm not going to potentially introduce new notices in the code by hastily updating the style of all of the HTML blocks in this code.

If you would like to see this change happen, feel free to fix it in a new pull request. There are tons of places in the older VideoPress codebase that have style choices that look like this that can be fixed. :)

**Comment by zinigor** [comment]
Thanks for fixing the whitespace! I agree, fixing all the other things can wait until the next PR.

**Pull request merged**